src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -122,6 +132,7 @@ public static boolean isRemoteStorageUrl(String path) {
      * @param path the path
      * @return an absolute file path if the original path was a relative file path, otherwise the original path
      */
+    //TODO: get rid of this..
```

Comment:
Let's burn this whole class with fire.

Reply:
Yes, but still more work to do to get there.